vuu-ui/packages/vuu-filters/src/filter-clause/useFilterClauseEditor.ts

```diff
@@ -160,24 +158,24 @@ export type FilterClauseCancelHandler = (
   reason: FilterClauseCancelType
 ) => void;
 export interface FilterClauseEditorHookProps {
+  columnDescriptors: Record<string, ColumnDescriptor>;
```

Comment:
maybe use the named type you defined for this

Reply:
Good point. Didn't have any type defined before, but now I do! `ColumnDescriptorsByName`